.editorconfig

```diff
@@ -7,6 +7,8 @@ indent_size = 4
 indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
+quote_type = single
+max_line_length = 120
```

Comment:
This may also be controversial but again, it kept most lines as they were and until we get styleci setup, it should help.